images/base/files/scripts/bootstrap_pipeline.py

```diff
@@ -41,16 +49,17 @@ def configure_cloud_run():
     run_id = os.environ["HEXA_RUN_ID"]
     workspace_slug = os.environ["HEXA_WORKSPACE"]
 
-    print("Downloading pipeline...")
-    os.mkdir("pipeline")
+    if not os.environ["HEXA_PIPELINE_NOTEBOOK"]:
```

Comment:
I think this `if` shows us that the configuration of the cloud env should not download the pipeline.

Reply:
?